.meta/sinks/aws_cloudwatch_logs.toml.erb

```diff
@@ -28,7 +28,7 @@ write_to_description = "[Amazon Web Service's CloudWatch Logs service][urls.aws_
 
 <%= render("_partials/fields/_component_options.toml", type: "sink", name: "aws_cloudwatch_logs") %>
 
-<%= render("_partials/fields/_batch_options.toml", namespace: "sinks.aws_cloudwatch_logs.options", common: false, max_events: 1000, max_size: nil, timeout_secs: 1) %>
+<%= render("_partials/fields/_batch_options.toml", namespace: "sinks.aws_cloudwatch_logs.options", common: false, max_bytes: nil, max_events: 1000, timeout_secs: 1) %>
```

Comment:
Here, the limits should be:

```
max_bytes: 1,048,576, max_events: 10000
```

As [described in the docs](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html). This is the most important sink for us and the one that originally raised the issue with our batch limits.

Reply:
Hmm, I missed this in the rest of the noise. Unfortunately, handling this adds some complication, and I think this PR is otherwise almost ready to merge. All of this work is a necessary prelude to handle this case. Can I drop the "closes" for that issue, merge this PR, and finish that work in a separate PR with this as a baseline to avoid dragging this one out further?

Reply:
That works for me.